inc/os_mock.h

```diff
@@ -4,6 +4,12 @@
 #ifndef _os_mock_h_
 #define _os_mock_h_
 
+#include "test_config.h" // Generated by cmake
```

Comment:
-D_HAVE_TIME might have been easier.  I have looked at this , and I think the simplest would be to #define CLOCK_MONOTONIC 4, which it is anyway, and typedef int clockid_t.  That passes for me on debian.